common/src/main/java/io/pravega/common/concurrent/FutureHelpers.java

```diff
@@ -109,6 +109,21 @@
     public static <T> boolean isSuccessful(CompletableFuture<T> f) {
         return f.isDone() && !f.isCompletedExceptionally() && !f.isCancelled();
     }
+    
+    /**
+     * If the future has failed returns the exception that caused it. Otherwise returns null.
```

Comment:
add a new line after this line. the following is the expectation.
```
/**
     * If the future has failed returns the exception that caused it. Otherwise returns null.
     *
     * @param future   The future to inspect.
     * @param <T> The Type of the future's result.
     * @return null or the exception that caused the Future to fail.
     */
```